python312Packages.tempora: 5.6.0 -> 5.7.0

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for python312Packages.tempora is: Objects and routines pertaining to date and time

meta.homepage for python312Packages.tempora is: https://github.com/jaraco/tempora

meta.changelog for python312Packages.tempora is: https://github.com/jaraco/tempora/blob/v5.7.0/NEWS.rst


###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes





###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 5.7.0 with grep in /nix/store/79c90mn2l6zdin6gx4wbiiml657qknpg-python3.12-tempora-5.7.0
- found 5.7.0 in filename of file in /nix/store/79c90mn2l6zdin6gx4wbiiml657qknpg-python3.12-tempora-5.7.0

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
73 total rebuild path(s)

73 package rebuild(s)

First fifty rebuilds by attrpath

adapta-gtk-theme
arc-theme
arx-libertatis
asciidoc-full
asciidoc-full-with-plugins
ayu-theme-gtk
breeze-hacked-cursor-theme
capitaine-cursors
catppuccin-cursors
ceph
ceph-client
ceph-csi
ceph-dev
cinnamon-screensaver
cinnamon.cinnamon-screensaver
cinnamon.iso-flags-png-320x420
clevis
coreboot-configurator
cwltool
dblatexFull
disorderfs
emojione
errbot
fava
fim
find-billy
flexget
frigate
hackneyed
heisenbridge
hikounomizu
home-assistant-component-tests.abode
inkscape
inkscape-extensions.textext
inkscape-with-extensions
iso-flags
iso-flags-png-320x240
k40-whisperer
kabeljau
kakounePlugins.rep
libceph
luksmeta
mate.mate-panel-with-applets
mate.mate-utils
material-cursors
mnemosyne
mojave-gtk-theme
numix-cursor-theme
numix-solarized-gtk-theme
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/79c90mn2l6zdin6gx4wbiiml657qknpg-python3.12-tempora-5.7.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A python312Packages.tempora https://github.com/r-ryantm/nixpkgs/archive/07ab7ef175c0764b644d240bf01fccde47fdc0b0.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/07ab7ef175c0764b644d240bf01fccde47fdc0b0#python312Packages.tempora
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/79c90mn2l6zdin6gx4wbiiml657qknpg-python3.12-tempora-5.7.0
ls -la /nix/store/79c90mn2l6zdin6gx4wbiiml657qknpg-python3.12-tempora-5.7.0/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>spring</li>
    <li>springLobby</li>
  </ul>
</details>
<details>
  <summary>155 packages failed to build:</summary>
  <ul>
    <li>adapta-gtk-theme</li>
    <li>arc-theme</li>
    <li>arx-libertatis</li>
    <li>asciidoc-full</li>
    <li>asciidoc-full-with-plugins</li>
    <li>asciidoc-full-with-plugins.dist</li>
    <li>asciidoc-full.dist</li>
    <li>ayu-theme-gtk</li>
    <li>breeze-hacked-cursor-theme</li>
    <li>capitaine-cursors</li>
    <li>catppuccin-cursors</li>
    <li>catppuccin-cursors.frappeBlue</li>
    <li>catppuccin-cursors.frappeDark</li>
    <li>catppuccin-cursors.frappeFlamingo</li>
    <li>catppuccin-cursors.frappeGreen</li>
    <li>catppuccin-cursors.frappeLavender</li>
    <li>catppuccin-cursors.frappeLight</li>
    <li>catppuccin-cursors.frappeMaroon</li>
    <li>catppuccin-cursors.frappeMauve</li>
    <li>catppuccin-cursors.frappePeach</li>
    <li>catppuccin-cursors.frappePink</li>
    <li>catppuccin-cursors.frappeRed</li>
    <li>catppuccin-cursors.frappeRosewater</li>
    <li>catppuccin-cursors.frappeSapphire</li>
    <li>catppuccin-cursors.frappeSky</li>
    <li>catppuccin-cursors.frappeTeal</li>
    <li>catppuccin-cursors.frappeYellow</li>
    <li>catppuccin-cursors.latteBlue</li>
    <li>catppuccin-cursors.latteDark</li>
    <li>catppuccin-cursors.latteFlamingo</li>
    <li>catppuccin-cursors.latteGreen</li>
    <li>catppuccin-cursors.latteLavender</li>
    <li>catppuccin-cursors.latteLight</li>
    <li>catppuccin-cursors.latteMaroon</li>
    <li>catppuccin-cursors.latteMauve</li>
    <li>catppuccin-cursors.lattePeach</li>
    <li>catppuccin-cursors.lattePink</li>
    <li>catppuccin-cursors.latteRed</li>
    <li>catppuccin-cursors.latteRosewater</li>
    <li>catppuccin-cursors.latteSapphire</li>
    <li>catppuccin-cursors.latteSky</li>
    <li>catppuccin-cursors.latteTeal</li>
    <li>catppuccin-cursors.latteYellow</li>
    <li>catppuccin-cursors.macchiatoBlue</li>
    <li>catppuccin-cursors.macchiatoDark</li>
    <li>catppuccin-cursors.macchiatoFlamingo</li>
    <li>catppuccin-cursors.macchiatoGreen</li>
    <li>catppuccin-cursors.macchiatoLavender</li>
    <li>catppuccin-cursors.macchiatoLight</li>
    <li>catppuccin-cursors.macchiatoMaroon</li>
    <li>catppuccin-cursors.macchiatoMauve</li>
    <li>catppuccin-cursors.macchiatoPeach</li>
    <li>catppuccin-cursors.macchiatoPink</li>
    <li>catppuccin-cursors.macchiatoRed</li>
    <li>catppuccin-cursors.macchiatoRosewater</li>
    <li>catppuccin-cursors.macchiatoSapphire</li>
    <li>catppuccin-cursors.macchiatoSky</li>
    <li>catppuccin-cursors.macchiatoTeal</li>
    <li>catppuccin-cursors.macchiatoYellow</li>
    <li>catppuccin-cursors.mochaBlue</li>
    <li>catppuccin-cursors.mochaDark</li>
    <li>catppuccin-cursors.mochaFlamingo</li>
    <li>catppuccin-cursors.mochaGreen</li>
    <li>catppuccin-cursors.mochaLavender</li>
    <li>catppuccin-cursors.mochaLight</li>
    <li>catppuccin-cursors.mochaMaroon</li>
    <li>catppuccin-cursors.mochaMauve</li>
    <li>catppuccin-cursors.mochaPeach</li>
    <li>catppuccin-cursors.mochaPink</li>
    <li>catppuccin-cursors.mochaRed</li>
    <li>catppuccin-cursors.mochaRosewater</li>
    <li>catppuccin-cursors.mochaSapphire</li>
    <li>catppuccin-cursors.mochaSky</li>
    <li>catppuccin-cursors.mochaTeal</li>
    <li>catppuccin-cursors.mochaYellow</li>
    <li>ceph</li>
    <li>ceph-client</li>
    <li>ceph-csi</li>
    <li>ceph.dev</li>
    <li>ceph.doc</li>
    <li>libceph (ceph.lib ,libceph.dev ,libceph.doc ,libceph.lib ,libceph.man)</li>
    <li>ceph.man</li>
    <li>cinnamon-screensaver</li>
    <li>clevis</li>
    <li>clevis.man</li>
    <li>coreboot-configurator</li>
    <li>cwltool</li>
    <li>cwltool.dist</li>
    <li>dblatexFull</li>
    <li>disorderfs</li>
    <li>emojione</li>
    <li>errbot</li>
    <li>errbot.dist</li>
    <li>fava</li>
    <li>fava.dist</li>
    <li>fim</li>
    <li>find-billy</li>
    <li>flexget</li>
    <li>flexget.dist</li>
    <li>frigate</li>
    <li>hackneyed</li>
    <li>heisenbridge</li>
    <li>heisenbridge.dist</li>
    <li>hikounomizu</li>
    <li>home-assistant-component-tests.abode</li>
    <li>inkscape</li>
    <li>inkscape-extensions.textext</li>
    <li>inkscape-extensions.textext.dist</li>
    <li>inkscape-with-extensions</li>
    <li>iso-flags</li>
    <li>iso-flags-png-320x240</li>
    <li>k40-whisperer</li>
    <li>kabeljau</li>
    <li>kakounePlugins.rep</li>
    <li>luksmeta</li>
    <li>mate.mate-panel-with-applets</li>
    <li>mate.mate-utils</li>
    <li>material-cursors</li>
    <li>mnemosyne</li>
    <li>mnemosyne.dist</li>
    <li>mojave-gtk-theme</li>
    <li>numix-cursor-theme</li>
    <li>numix-solarized-gtk-theme</li>
    <li>pip-audit</li>
    <li>pip-audit.dist</li>
    <li>plata-theme</li>
    <li>poetry</li>
    <li>poetry.dist</li>
    <li>poetryPlugins.poetry-audit-plugin</li>
    <li>poetryPlugins.poetry-audit-plugin.dist</li>
    <li>poetryPlugins.poetry-plugin-up</li>
    <li>poetryPlugins.poetry-plugin-up.dist</li>
    <li>pop-gtk-theme</li>
    <li>powerhub</li>
    <li>powerhub.dist</li>
    <li>pyload-ng</li>
    <li>pyload-ng.dist</li>
    <li>qemu_full</li>
    <li>qemu_full.debug</li>
    <li>qemu_full.ga</li>
    <li>quickemu</li>
    <li>quickgui</li>
    <li>sabnzbd</li>
    <li>samba4Full</li>
    <li>samba4Full.dev</li>
    <li>samba4Full.man</li>
    <li>sherlock</li>
    <li>sherlock.dist</li>
    <li>snapcraft</li>
    <li>snapcraft.dist</li>
    <li>tang</li>
    <li>tang.man</li>
    <li>vimix-cursors</li>
    <li>volantes-cursors</li>
    <li>wpa_supplicant_gui</li>
  </ul>
</details>

---

###### Maintainer pings

dotlambda commented on pkgs/development/python-modules/tempora/default.nix:
```diff
@@ -13,14 +13,14 @@
   pname = "tempora";
-  version = "5.6.0";
+  version = "5.7.0";
   format = "pyproject";
```
```suggestion
  pyproject = true;
```

dotlambda commented on pkgs/development/python-modules/tempora/default.nix:
```diff
@@ -13,14 +13,14 @@
+    hash = "sha256-iIGQotvjJV/ybfqfzssl9NOENMDxlDzWHemLtBxBDFA=";
   };
 
   nativeBuildInputs = [ setuptools-scm ];
```
```suggestion
  build-system = [ setuptools-scm ];
```

dotlambda committed:
python312Packages.tempora: 5.6.0 -> 5.7.0

Diff: https://github.com/jaraco/tempora/compare/refs/tags/v5.6.0...v5.7.0

Changelog: https://github.com/jaraco/tempora/blob/v5.7.0/NEWS.rst

natsukium commented:
Result of `nixpkgs-review pr 330141` at d2db417d run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>spring</li>
    <li>springLobby</li>
  </ul>
</details>
<details>
  <summary>16 packages failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/natsukium/fb1304ade2a05cefb57d0f581ffde438">ceph</a> (ceph-dev)</li>
    <li>ceph-client</li>
    <li>ceph-csi</li>
    <li><a href="https://gist.github.com/natsukium/ba8ee981df21234fb990f193a760da97">ceph.dev</a> (ceph-dev.dev)</li>
    <li><a href="https://gist.github.com/natsukium/a928f83abe3b856453e35fc44b1986a6">ceph.doc</a> (ceph-dev.doc)</li>
    <li><a href="https://gist.github.com/natsukium/4acc3d132dd880e846dc0c60868636d7">libceph</a> (ceph.lib ,libceph.dev ,libceph.doc ,libceph.lib ,libceph.man)</li>
    <li><a href="https://gist.github.com/natsukium/65b6cf1522d7b37655a36a6146f0bac7">ceph.man</a> (ceph-dev.man)</li>
    <li><a href="https://gist.github.com/natsukium/910d4f37456e2d786e79b43041e9a1d2">flexget</a></li>
    <li>qemu_full</li>
    <li>qemu_full.debug</li>
    <li>qemu_full.ga</li>
    <li>quickemu</li>
    <li>quickgui</li>
    <li>sambaFull (samba4Full)</li>
    <li>sambaFull.dev (samba4Full.dev)</li>
    <li>sambaFull.man (samba4Full.man)</li>
  </ul>
</details>
<details>
  <summary>122 packages built successfully:</summary>
  <ul>
    <li>adapta-gtk-theme</li>
    <li>arc-theme</li>
    <li>arx-libertatis</li>
    <li>asciidoc-full</li>
    <li>asciidoc-full-with-plugins</li>
    <li>ayu-theme-gtk</li>
    <li>breeze-hacked-cursor-theme</li>
    <li>capitaine-cursors</li>
    <li>catppuccin-cursors</li>
    <li>catppuccin-cursors.frappeBlue</li>
    <li>catppuccin-cursors.frappeDark</li>
    <li>catppuccin-cursors.frappeFlamingo</li>
    <li>catppuccin-cursors.frappeGreen</li>
    <li>catppuccin-cursors.frappeLavender</li>
    <li>catppuccin-cursors.frappeLight</li>
    <li>catppuccin-cursors.frappeMaroon</li>
    <li>catppuccin-cursors.frappeMauve</li>
    <li>catppuccin-cursors.frappePeach</li>
    <li>catppuccin-cursors.frappePink</li>
    <li>catppuccin-cursors.frappeRed</li>
    <li>catppuccin-cursors.frappeRosewater</li>
    <li>catppuccin-cursors.frappeSapphire</li>
    <li>catppuccin-cursors.frappeSky</li>
    <li>catppuccin-cursors.frappeTeal</li>
    <li>catppuccin-cursors.frappeYellow</li>
    <li>catppuccin-cursors.latteBlue</li>
    <li>catppuccin-cursors.latteDark</li>
    <li>catppuccin-cursors.latteFlamingo</li>
    <li>catppuccin-cursors.latteGreen</li>
    <li>catppuccin-cursors.latteLavender</li>
    <li>catppuccin-cursors.latteLight</li>
    <li>catppuccin-cursors.latteMaroon</li>
    <li>catppuccin-cursors.latteMauve</li>
    <li>catppuccin-cursors.lattePeach</li>
    <li>catppuccin-cursors.lattePink</li>
    <li>catppuccin-cursors.latteRed</li>
    <li>catppuccin-cursors.latteRosewater</li>
    <li>catppuccin-cursors.latteSapphire</li>
    <li>catppuccin-cursors.latteSky</li>
    <li>catppuccin-cursors.latteTeal</li>
    <li>catppuccin-cursors.latteYellow</li>
    <li>catppuccin-cursors.macchiatoBlue</li>
    <li>catppuccin-cursors.macchiatoDark</li>
    <li>catppuccin-cursors.macchiatoFlamingo</li>
    <li>catppuccin-cursors.macchiatoGreen</li>
    <li>catppuccin-cursors.macchiatoLavender</li>
    <li>catppuccin-cursors.macchiatoLight</li>
    <li>catppuccin-cursors.macchiatoMaroon</li>
    <li>catppuccin-cursors.macchiatoMauve</li>
    <li>catppuccin-cursors.macchiatoPeach</li>
    <li>catppuccin-cursors.macchiatoPink</li>
    <li>catppuccin-cursors.macchiatoRed</li>
    <li>catppuccin-cursors.macchiatoRosewater</li>
    <li>catppuccin-cursors.macchiatoSapphire</li>
    <li>catppuccin-cursors.macchiatoSky</li>
    <li>catppuccin-cursors.macchiatoTeal</li>
    <li>catppuccin-cursors.macchiatoYellow</li>
    <li>catppuccin-cursors.mochaBlue</li>
    <li>catppuccin-cursors.mochaDark</li>
    <li>catppuccin-cursors.mochaFlamingo</li>
    <li>catppuccin-cursors.mochaGreen</li>
    <li>catppuccin-cursors.mochaLavender</li>
    <li>catppuccin-cursors.mochaLight</li>
    <li>catppuccin-cursors.mochaMaroon</li>
    <li>catppuccin-cursors.mochaMauve</li>
    <li>catppuccin-cursors.mochaPeach</li>
    <li>catppuccin-cursors.mochaPink</li>
    <li>catppuccin-cursors.mochaRed</li>
    <li>catppuccin-cursors.mochaRosewater</li>
    <li>catppuccin-cursors.mochaSapphire</li>
    <li>catppuccin-cursors.mochaSky</li>
    <li>catppuccin-cursors.mochaTeal</li>
    <li>catppuccin-cursors.mochaYellow</li>
    <li>cinnamon-screensaver</li>
    <li>clevis</li>
    <li>clevis.man</li>
    <li>coreboot-configurator</li>
    <li>cwltool</li>
    <li>dblatexFull</li>
    <li>disorderfs</li>
    <li>emojione</li>
    <li>errbot</li>
    <li>fava</li>
    <li>fim</li>
    <li>find-billy</li>
    <li>frigate</li>
    <li>hackneyed</li>
    <li>heisenbridge</li>
    <li>hikounomizu</li>
    <li>home-assistant-component-tests.abode</li>
    <li>inkscape</li>
    <li>inkscape-extensions.textext</li>
    <li>inkscape-with-extensions</li>
    <li>iso-flags</li>
    <li>iso-flags-png-320x240</li>
    <li>k40-whisperer</li>
    <li>kabeljau</li>
    <li>rep (kakounePlugins.rep)</li>
    <li>luksmeta</li>
    <li>mate.mate-panel-with-applets</li>
    <li>mate.mate-utils</li>
    <li>material-cursors</li>
    <li>mnemosyne</li>
    <li>mojave-gtk-theme</li>
    <li>numix-cursor-theme</li>
    <li>numix-solarized-gtk-theme</li>
    <li>pip-audit</li>
    <li>plata-theme</li>
    <li>poetry</li>
    <li>poetryPlugins.poetry-audit-plugin</li>
    <li>poetryPlugins.poetry-plugin-up</li>
    <li>pop-gtk-theme</li>
    <li>powerhub</li>
    <li>pyload-ng</li>
    <li>sabnzbd</li>
    <li>sherlock</li>
    <li>snapcraft</li>
    <li>tang</li>
    <li>tang.man</li>
    <li>vimix-cursors</li>
    <li>volantes-cursors</li>
    <li>wpa_supplicant_gui</li>
  </ul>
</details>

wegank commented:
Result of `nixpkgs-review pr 330141` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>spring</li>
    <li>springLobby</li>
  </ul>
</details>
<details>
  <summary>3 packages failed to build:</summary>
  <ul>
    <li>quickgui</li>
    <li>quickgui.debug</li>
    <li>quickgui.pubcache</li>
  </ul>
</details>
<details>
  <summary>154 packages built:</summary>
  <ul>
    <li>adapta-gtk-theme</li>
    <li>arc-theme</li>
    <li>arx-libertatis</li>
    <li>asciidoc-full</li>
    <li>asciidoc-full-with-plugins</li>
    <li>asciidoc-full-with-plugins.dist</li>
    <li>asciidoc-full.dist</li>
    <li>ayu-theme-gtk</li>
    <li>breeze-hacked-cursor-theme</li>
    <li>capitaine-cursors</li>
    <li>catppuccin-cursors</li>
    <li>catppuccin-cursors.frappeBlue</li>
    <li>catppuccin-cursors.frappeDark</li>
    <li>catppuccin-cursors.frappeFlamingo</li>
    <li>catppuccin-cursors.frappeGreen</li>
    <li>catppuccin-cursors.frappeLavender</li>
    <li>catppuccin-cursors.frappeLight</li>
    <li>catppuccin-cursors.frappeMaroon</li>
    <li>catppuccin-cursors.frappeMauve</li>
    <li>catppuccin-cursors.frappePeach</li>
    <li>catppuccin-cursors.frappePink</li>
    <li>catppuccin-cursors.frappeRed</li>
    <li>catppuccin-cursors.frappeRosewater</li>
    <li>catppuccin-cursors.frappeSapphire</li>
    <li>catppuccin-cursors.frappeSky</li>
    <li>catppuccin-cursors.frappeTeal</li>
    <li>catppuccin-cursors.frappeYellow</li>
    <li>catppuccin-cursors.latteBlue</li>
    <li>catppuccin-cursors.latteDark</li>
    <li>catppuccin-cursors.latteFlamingo</li>
    <li>catppuccin-cursors.latteGreen</li>
    <li>catppuccin-cursors.latteLavender</li>
    <li>catppuccin-cursors.latteLight</li>
    <li>catppuccin-cursors.latteMaroon</li>
    <li>catppuccin-cursors.latteMauve</li>
    <li>catppuccin-cursors.lattePeach</li>
    <li>catppuccin-cursors.lattePink</li>
    <li>catppuccin-cursors.latteRed</li>
    <li>catppuccin-cursors.latteRosewater</li>
    <li>catppuccin-cursors.latteSapphire</li>
    <li>catppuccin-cursors.latteSky</li>
    <li>catppuccin-cursors.latteTeal</li>
    <li>catppuccin-cursors.latteYellow</li>
    <li>catppuccin-cursors.macchiatoBlue</li>
    <li>catppuccin-cursors.macchiatoDark</li>
    <li>catppuccin-cursors.macchiatoFlamingo</li>
    <li>catppuccin-cursors.macchiatoGreen</li>
    <li>catppuccin-cursors.macchiatoLavender</li>
    <li>catppuccin-cursors.macchiatoLight</li>
    <li>catppuccin-cursors.macchiatoMaroon</li>
    <li>catppuccin-cursors.macchiatoMauve</li>
    <li>catppuccin-cursors.macchiatoPeach</li>
    <li>catppuccin-cursors.macchiatoPink</li>
    <li>catppuccin-cursors.macchiatoRed</li>
    <li>catppuccin-cursors.macchiatoRosewater</li>
    <li>catppuccin-cursors.macchiatoSapphire</li>
    <li>catppuccin-cursors.macchiatoSky</li>
    <li>catppuccin-cursors.macchiatoTeal</li>
    <li>catppuccin-cursors.macchiatoYellow</li>
    <li>catppuccin-cursors.mochaBlue</li>
    <li>catppuccin-cursors.mochaDark</li>
    <li>catppuccin-cursors.mochaFlamingo</li>
    <li>catppuccin-cursors.mochaGreen</li>
    <li>catppuccin-cursors.mochaLavender</li>
    <li>catppuccin-cursors.mochaLight</li>
    <li>catppuccin-cursors.mochaMaroon</li>
    <li>catppuccin-cursors.mochaMauve</li>
    <li>catppuccin-cursors.mochaPeach</li>
    <li>catppuccin-cursors.mochaPink</li>
    <li>catppuccin-cursors.mochaRed</li>
    <li>catppuccin-cursors.mochaRosewater</li>
    <li>catppuccin-cursors.mochaSapphire</li>
    <li>catppuccin-cursors.mochaSky</li>
    <li>catppuccin-cursors.mochaTeal</li>
    <li>catppuccin-cursors.mochaYellow</li>
    <li>ceph (ceph-dev)</li>
    <li>ceph-client</li>
    <li>ceph-csi</li>
    <li>ceph.dev (ceph-dev.dev)</li>
    <li>ceph.doc (ceph-dev.doc)</li>
    <li>libceph (ceph.lib ,libceph.dev ,libceph.doc ,libceph.lib ,libceph.man)</li>
    <li>ceph.man (ceph-dev.man)</li>
    <li>cinnamon-screensaver</li>
    <li>clevis</li>
    <li>clevis.man</li>
    <li>coreboot-configurator</li>
    <li>cwltool</li>
    <li>cwltool.dist</li>
    <li>dblatexFull</li>
    <li>disorderfs</li>
    <li>emojione</li>
    <li>errbot</li>
    <li>errbot.dist</li>
    <li>fava</li>
    <li>fava.dist</li>
    <li>fim</li>
    <li>find-billy</li>
    <li>flexget</li>
    <li>flexget.dist</li>
    <li>frigate</li>
    <li>hackneyed</li>
    <li>heisenbridge</li>
    <li>heisenbridge.dist</li>
    <li>hikounomizu</li>
    <li>home-assistant-component-tests.abode</li>
    <li>inkscape</li>
    <li>inkscape-extensions.textext</li>
    <li>inkscape-extensions.textext.dist</li>
    <li>inkscape-with-extensions</li>
    <li>iso-flags</li>
    <li>iso-flags-png-320x240</li>
    <li>k40-whisperer</li>
    <li>kabeljau</li>
    <li>rep (kakounePlugins.rep)</li>
    <li>luksmeta</li>
    <li>mate.mate-panel-with-applets</li>
    <li>mate.mate-utils</li>
    <li>material-cursors</li>
    <li>mnemosyne</li>
    <li>mnemosyne.dist</li>
    <li>mojave-gtk-theme</li>
    <li>numix-cursor-theme</li>
    <li>numix-solarized-gtk-theme</li>
    <li>pip-audit</li>
    <li>pip-audit.dist</li>
    <li>plata-theme</li>
    <li>poetry</li>
    <li>poetry.dist</li>
    <li>poetryPlugins.poetry-audit-plugin</li>
    <li>poetryPlugins.poetry-audit-plugin.dist</li>
    <li>poetryPlugins.poetry-plugin-up</li>
    <li>poetryPlugins.poetry-plugin-up.dist</li>
    <li>pop-gtk-theme</li>
    <li>powerhub</li>
    <li>powerhub.dist</li>
    <li>pyload-ng</li>
    <li>pyload-ng.dist</li>
    <li>qemu_full</li>
    <li>qemu_full.debug</li>
    <li>qemu_full.ga</li>
    <li>quickemu</li>
    <li>sabnzbd</li>
    <li>sambaFull (samba4Full)</li>
    <li>sambaFull.dev (samba4Full.dev)</li>
    <li>sambaFull.man (samba4Full.man)</li>
    <li>sherlock</li>
    <li>sherlock.dist</li>
    <li>snapcraft</li>
    <li>snapcraft.dist</li>
    <li>tang</li>
    <li>tang.man</li>
    <li>vimix-cursors</li>
    <li>volantes-cursors</li>
    <li>wpa_supplicant_gui</li>
  </ul>
</details>
